src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,82 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static final int INVALID_AGE = 17;
+    private static final String INVALID_SMALL_PASSWORD = "12345";
+    private static final int VALID_AGE = 19;
+    private static final String VALID_NEW_LOGIN = "Kittycat@gmail.com";
+    private static final String VALID_PASSWORD = "67hgj78";
+    private static RegistrationServiceImpl registrationServiceImpl;
+    private static StorageDao storageDao;
+    private static User user;
+
+    @BeforeAll
+    static void beforeAll() {
+        registrationServiceImpl = new RegistrationServiceImpl();
+        storageDao = new StorageDaoImpl();
+        user = new User();
+    }
+
+    @Test
+    void register_nullAge_isNotOk() {
+        user.setAge(null);
+        assertThrows(RuntimeException.class, () -> registrationServiceImpl.register(user));
+    }
+
+    @Test
+    void register_smallAge_isNotOk() {
+        user.setAge(INVALID_AGE);
+        assertThrows(RuntimeException.class, () -> registrationServiceImpl.register(user));
+    }
+
+    @Test
+    void register_existingLogin_isNotOk() {
+        user.setLogin(user.getLogin());
+        assertThrows(RuntimeException.class, () -> registrationServiceImpl.register(user));
+    }
```

Comment:
try to check the message here
you can do it this way
<img width="474" alt="image" src="https://user-images.githubusercontent.com/58032907/198005482-4e585303-a3cd-4b21-983c-64a2158c767e.png">
and try to figure out why you see that message